core/controllers/feedback_test.py

```diff
@@ -102,14 +101,15 @@ def test_non_logged_in_users_cannot_create_threads_and_messages(self):
             '%s/%s' % (
                 feconf.FEEDBACK_THREADLIST_URL_PREFIX, self.EXP_ID),
             {
-                'state_name': 'Welcome!',
                 'subject': self.UNICODE_TEST_STRING,
                 'text': self.UNICODE_TEST_STRING,
             }, csrf_token=self.csrf_token,
             expect_errors=True, expected_status_int=401)
 
-        thread_url = '%s/%s' % (
-            feconf.FEEDBACK_THREAD_URL_PREFIX, '0.dummy_thread_id')
+        with self.swap(feconf, 'ENABLE_GENERALIZED_FEEDBACK_THREADS', True):
```

Comment:
why do you need self.swap here? I don't see anything that's dependent on a function.